fulp_modules/main_features/bloodsuckers/code/mobs/clawmonster.dm

```diff
@@ -0,0 +1,14 @@
+/mob/living/simple_animal/hostile/retaliate/tzimisce/clawmonster
+	name = "clawed monster"
+	icon_state = "tarantula"
+	icon_living = "tarantula"
+	icon_dead = "tarantula_dead"
+	maxHealth = 100
+	health = 100
+	melee_damage_lower = 20
+	melee_damage_upper = 25
+	speed = 0.5
+	damage_coeff = list(BRUTE = 0.3, BURN = 0.9, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 1)
```

Comment:
holy fuck these things have an effective 333+ max health vs. brute attacks

if you don't have a laser or a welder, you're just kind of fucked